playbooks/ansible-cloud/appliance-base/pre.yaml

```diff
@@ -52,3 +52,10 @@
         write_inventory_dest: /etc/ansible/hosts
         write_inventory_exclude_hostvars:
           - ansible_connection
+
+    - name: Add a vcenter entry in /etc/hosts
+      lineinfile:
+        dest: /etc/hosts
+        regexp: " vcenter "
+        line: "{{ hostvars['vmware-vcsa-6.7.0']['nodepool']['private_ipv4']}} vcenter.test vcenter"
```

Comment:
same, ansible_host here. not private

Reply:
ok. Should be addressed in the next update.